### PR TITLE
Refactor nxos_reboot module

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
* Minor PEP8 fixes
* Removed un-used code

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/network/nxos/nxos_reboot.py

##### ANSIBLE VERSION
```
2.4 devel
```

### CODE REVIEW

File: lib/ansible/modules/network/nxos/nxos_reboot.py [hunk 1]
```diff
@@ -86,7 +87,7 @@ def main():
     changed = True
 
     results = {
-        'changed': True,
+        'changed': changed,
```
Comment (trishnaguha) [inline lib/ansible/modules/network/nxos/nxos_reboot.py]: Can you remove this block of code [L89-L92](https://github.com/Akasurde/ansible/blob/fd1e79d33e03ee10e78f512f04954ae29e9a451e/lib/ansible/modules/network/nxos/nxos_reboot.py#L89-L92)? And we can have `results` initialized like `dict(changed=False, warnings=warnings)` instead?
In that case `changed = True` should also be updated to `results['changed'] = True`.

**Review by trishnaguha — CHANGES_REQUESTED** [review]

**Review by trishnaguha — COMMENTED** [review]
Hi @Akasurde , thanks for the PR.
Some changes are requested for the PR and we are good to go after that.
This is the pattern we are trying to follow.

**Comment by Akasurde** [comment]
@trishnaguha Done. Have a look.

**Review by trishnaguha — APPROVED** [review]

**Comment by Akasurde** [comment]
@trishnaguha Thanks for review and comments.

**Pull request merged**